apis/sgv2-restapi/pom.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <!-- Parent -->
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>sgv2-api-parent</artifactId>
     <version>2.0.10-SNAPSHOT</version>
   </parent>
-  <groupId>io.stargate.web</groupId>
```

Comment:
I moved group back to the `io.stargate`, I see no reason for a difference here.. Since we never publish this artifact, I guess it makes no difference

Reply:
+1